tests/integration/api/v1/contents/get.test.js

```diff
@@ -1246,5 +1246,172 @@ describe('GET /api/v1/contents', () => {
       expect(uuidVersion(responseBody.error_id)).toEqual(4);
       expect(uuidVersion(responseBody.request_id)).toEqual(4);
     });
+
+    describe('Content type query params', () => {
+      test.each([
+        { content: 'relevant root', params: '' },
+        { content: 'relevant root, including draft,', params: 'with_children=false' },
+        { content: 'relevant children', params: 'with_children=true' },
+        { content: 'new root, including draft,', params: 'with_children=false&with_root=true&strategy=new' },
+        { content: 'new root', params: 'with_children=false&with_root=true&strategy=new' },
+        { content: 'new root', params: 'with_children=false&strategy=new' },
+        { content: 'new children', params: 'with_children=true&with_root=false&strategy=new' },
+        { content: 'new root and children', params: 'with_children=true&strategy=new' },
+        { content: 'new root and children', params: 'with_children=true&with_root=true&strategy=new' },
+        { content: 'new root, including draft,', params: 'with_root=true&strategy=new' },
```

Comment:
@Rafatcb Não seria melhor colocar os params em uma array?

```js
        { content: 'relevant root', params: [] },
        { content: 'relevant root, including draft,', params: ['with_children=false'] },
        { content: 'relevant children', params: ['with_children=true'] },
        { content: 'new root, including draft,', params: ['with_children=false', 'with_root=true', 'strategy=new'] },
        { content: 'new root', params: ['with_children=false', 'with_root=true', 'strategy=new'] },
        { content: 'new root', params: ['with_children=false', 'strategy=new'] },
        { content: 'new children', params: ['with_children=true', 'with_root=false', 'strategy=new'] },
        { content: 'new root and children', params: ['with_children=true', 'strategy=new'] },
        { content: 'new root and children', params: ['with_children=true', 'with_root=true', 'strategy=new'] },
        { content: 'new root, including draft,', params: ['with_root=true', 'strategy=new'] },
```

Reply:
Não tenho preferência quanto a isso. Acho sua sugestão facilita a criação de novos testes, mas o título dos testes piora um pouco, o que não é nada grave. Se for fazer essa alteração, talvez seja melhor atualizar o título do teste para `get $content with params: $params`, já que um array "tira a cara" de parâmetros de URL. O resultado ficaria assim:

```sh
Content type query params
  ✓ get relevant root with params: [] (648 ms)
  ✓ get relevant root, including draft, with params: ["with_children=false"] (504 ms)
  ✓ get relevant children with params: ["with_children=true"] (618 ms)
  ✓ get new root, including draft, with params: ["with_children=false", "with_root=true", "strategy=new"] (506 ms)
  ✓ get new root with params: ["with_children=false", "with_root=true", "strategy=new"] (503 ms)
  ✓ get new root with params: ["with_children=false", "strategy=new"] (498 ms)
  ✓ get new children with params: ["with_children=true", "with_root=false", "strategy=new"] (554 ms)
  ✓ get new root and children with params: ["with_children=true", "strategy=new"] (565 ms)
  ✓ get new root and children with params: ["with_children=true", "with_root=true", "strategy=new"] (497 ms)
  ✓ get new root, including draft, with params: ["with_root=true", "strategy=new"] (473 ms)
```

Dessa forma, os títulos não me parece ruins, apesar de ter um array.

Por mim, se você quiser, pode fazer a alteração :+1:

Reply:
@Rafatcb É apenas para ter um visual melhor do que estão nos params, do que eles tudo juntos.

Reply:
Então faz essa alteração, faz o rebase (ao invés do merge) e já deixa tudo num commit, que o @aprendendofelipe aprovando já fica possível de realizar o merge do PR.

Reply:
@Rafatcb feito!